### PR TITLE
Add documentation as the main README

### PR DESCRIPTION
**Pull request by CPatchane**
Add all documentation needed to use the registry and publish an application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,468 @@
...
+...
+registries:
+  default:
+    - http://localhost:8081/
+...
+```
+
+Now restart your local `cozy-stack` to take this new configuration in consideration (stop and run again `cozy-stack serve`) and you're ready to work with the `cozy-apps-registry`!
+
+## Publish your application on the registry
```
Comment (nono) [inline README.md]: I'd prefer if the `Publish your application on the registry` can be moved to a tutorial on https://github.com/cozy/cozy-docs-v3

Comment (CPatchane) [reply README.md]: Agree and keep a link in this README to have an entry from here as well 👍

File: README.md [hunk 2]
```diff
@@ -1,57 +1,470 @@
...
+Before starting, you will need to have a couchdb running already. That can be the one used by the local `cozy-stack` if you have one. For this tutorial, couchdb will be running on the default port 5984.
+
+### 1) Install and configure the local `cozy-apps-registry`
+
+Since this is a golang project, you can install it using `go` with the followed command:
+
+```shell
+go get -u github.com/cozy/cozy-apps-registry
+go install
+cd $GOPATH/src/github.com/cozy/cozy-apps-registry/
```
[the remark this answers is not in the document]
Comment (kosssi) [reply README.md]: @nono un `go get ...` ne fait pas automatiquement un `go install` ?

**Comment by kosssi** [comment]
j'imagine que tout la partie `Travis` changera avec `cozy/cozy-app-publish`

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nono) [reply README.md]: Oui, pas besoin d'un `go install` après un `go get`

File: README.md [hunk 3]
```diff
@@ -1,57 +1,470 @@
...
+
+Let's add the Drive application with the previous manifest as example:
+
+```shell
+# {{EDITOR_TOKEN}} -> your generated editor access token
+curl -X "POST" "http://localhost:8081/registry" \
+     -H "Authorization: Token {{EDITOR_TOKEN}}" \
+     -H "Content-Type: application/json" \
+     -d $'{
+  "slug": "drive",
```
[the remark this answers is not in the document]
Comment (gregorylegarec) [reply README.md]: Precision : In the drive release, the version differs between the `manifest.webapp` and the `package.json`. That's why it fails for the current releases and cannot be used as example in this README.

**Comment by CPatchane** [comment]
Updated 👍 
I will add a link to this documentation from our official docs https://github.com/cozy/cozy-docs-v3

**Comment by CPatchane** [comment]
[(5) Publication des applications et connecteurs externes](https://trello.com/c/QiJZL5N7/939-5-publication-des-applications-et-connecteurs-externes)

**Pull request merged**